src/common-events/components/rangeDropdown/__tests__/RangeDropdown.test.tsx

```diff
@@ -144,10 +144,14 @@ test('can be navigated with tab', async () => {
   }) as HTMLInputElement;
 
   userEvent.tab();
```

Comment:
Because of failed CI run
https://github.com/City-of-Helsinki/hobbies-helsinki/actions/runs/3060252516/jobs/4938539706

I assumed these are related to upgraded test modules